app/src/main/java/com/example/makeiteven2/game/Game.kt

```diff
@@ -2,7 +2,11 @@ package com.example.makeiteven2.game
 
 import android.util.Log
 import android.widget.ToggleButton
+import com.example.makeiteven2.extras.EOperators
```

Comment:
can you mark all the code of test in some way to distinguish between test and real